network_observability/observing-network-traffic.adoc

```diff
@@ -26,6 +26,12 @@ include::modules/network-observability-dns-overview.adoc[leveloffset=+2]
 * xref:../network_observability/observing-network-traffic.adoc#network-observability-dns-tracking_nw-observe-network-traffic[Working with DNS tracking]
 * xref:../network_observability/metrics-alerts-dashboards.adoc#network-observability-metrics_metrics-dashboards-alerts[Network Observability metrics]
 
+include::modules/network-observability-RTT-overview.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../network_observability/observing-network-traffic.adoc#network-observability-RTT_nw-observe-network-traffic[Working with RTT tracing]
+
```

Comment:
This was accidentally removed in a rebase of this[PR](https://github.com/openshift/openshift-docs/pull/69106/files#diff-21cba4b9eea6286345ee8e2ad7b3cc0e4dc6ac125a47623a7df9f52f2f2506aaL27-L31)

Reply:
I merged another PR to add this back in.